### PR TITLE
Getting started dependencies, and updated platforms doc to include browsers/OS

### PR DESCRIPTION
**Pull request by illume**
This should help people to get started by telling them the dependencies.

### CODE REVIEW

File: docs/development/_index.md [hunk 1]
```diff
@@ -7,10 +7,20 @@ This is a quickstart guide for building and running Headlamp for development.
...
 
+See [platforms](../platforms.md) to find out which browsers, OS and flavours of Kubernetes we support.
+
+## Dependencies to get started
+
+These are the required dependencies to get started. Other dependencies are pulled in by the golang or node package managers (see frontend/package.json, app/package.json, backend/go.mod and Dockerfile).
+
+- [Node.js](https://nodejs.org/en/download/) Latest LTS (18.14 at time of writing). Many of us use [nvm](https://github.com/nvm-sh/nvm) for installing multiple versions of Node.
+- [Go](https://go.dev/doc/install), (1.19 at time of writing)
+- Make (GNU). On Windows this can be installed with the "chocolatey" package manager that is installed with node.
```
Comment (ashu8912) [inline docs/development/_index.md:18]: should we add a link to make website too https://www.gnu.org/software/make/ ?? WDYT??

File: docs/development/_index.md [hunk 2]
```diff
@@ -7,10 +7,20 @@ This is a quickstart guide for building and running Headlamp for development.
...
+- Kubernetes, we suggest [minikube](https://minikube.sigs.k8s.io/docs/) as one good K8s installation for testing locally. Other k8s installations are supported (see [platforms](../platforms.md)).
```
Comment (ashu8912) [inline docs/development/_index.md:19]: Same can the kubernetes term here be linked to the kubernetes website??

File: docs/development/_index.md (hunk 1 above)
Comment (illume) [reply docs/development/_index.md:18]: Good idea.

File: docs/development/_index.md (hunk 2 above)
Comment (illume) [reply docs/development/_index.md]: Thanks. Done.

**Pull request merged**